base/io.jl

```diff
@@ -65,13 +65,55 @@ function readavailable end
     isreadable(io) -> Bool
 
 Return `true` if the specified IO object is readable (if that can be determined).
+
+# Examples
+```jldoctest
+julia> io = open("myfile.txt", "w");
```

Comment:
Should we write these with the `do` syntax?

Reply:
can do :)